examples/deepdds_examples.py

```diff
@@ -0,0 +1,26 @@
+"""Example with DeepDDs."""
+
+from chemicalx import pipeline
+from chemicalx.models import DeepDDS
+
+
+def main():
+    """Train and evaluate the DeepDDs model."""
+    model = DeepDDS(
```

Comment:
follow pattern from other examples for pre-loading dataset and instantiating the model based on its properties